### DISCUSSION TITLE
Beware fraudulent site posing as Rectangle

### DISCUSSION THREAD

**Discussion by rxhanson** [rxhanson/Rectangle#1092] [General]
Last year, someone created fraudulent site posing as Rectangle's site that tries to sell users a cracked version of Rectangle Pro. With your help, I was able to get enough people to report it to the domain registrar to pull it down. It's back up under a different URL, and if you have a few minutes, I would appreciate if you report it.

The fraudulent URL:
rectanglemacos(dot)com

All the places to report:

Google safe browsing
https://safebrowsing.google.com/safebrowsing/report_badware/
https://safebrowsing.google.com/safebrowsing/report_phish/

Cloudflare
https://abuse.cloudflare.com/

Microsoft Security Intelligence
https://www.microsoft.com/en-us/wdsi/support/report-unsafe-site-guest

DuckDuckGo
Send an email to [abuse@duckduckgo.com](mailto:abuse@duckduckgo.com).

Dynadot (the registrar)
https://www.dynadot.com/report-abuse.html

Twitter: fraudulent handle is @Rectanglemac
https://help.twitter.com/en/forms/authenticity/impersonation

If you fell for the scam and paid via PayPal, please report the fraud and get your money back.

Thanks!

**Comment by lpridpath** [comment]
Submitted a report on all linked sites. Sorry this is happening again - hope it gets taken down soon.

Best.

**Comment by ylectric** [comment]
I also submitted a report on all linked sites. Seems like none of those places are doing anything about it.

Beyond that, I investigated the fraudulent site and found that this is the person behind it:

Shamin Madhuranga
Email: shaminmadhuranga@gmail.com
Whatsapp: +17573678550
Phone: +94742094558
Lives in Colombo, Sri Lanka

**Comment by porg** [comment]
Using unlicensed software per se is already unfriendly. And then selling stolen/cracked software is really disgusting. Glad that you could get that imposter website down!
